utils_mysql.go

```diff
@@ -40,38 +41,40 @@ func GetColumnsFromMysqlTable(mariadbUser string, mariadbPassword string, mariad
 
 	if err != nil {
 		fmt.Println("Error selecting from db: " + err.Error())
-		return nil, err
+		return nil, nil, err
 	}
 	if rows != nil {
 		defer rows.Close()
 	} else {
-		return nil, errors.New("No results returned for table")
+		return nil, nil, errors.New("No results returned for table")
 	}
 
 	for rows.Next() {
 		var column string
 		var columnKey string
 		var dataType string
 		var nullable string
-		rows.Scan(&column, &columnKey, &dataType, &nullable)
+		var comment string
+		rows.Scan(&column, &columnKey, &dataType, &nullable, &comment)
 
-		columnDataTypes[column] = map[string]string{"value": dataType, "nullable": nullable, "primary": columnKey}
+		columnDataTypes[column] = map[string]string{"value": dataType, "nullable": nullable, "primary": columnKey, "comment": comment}
+		columnNamesSorted = append(columnNamesSorted, column)
 	}
 
-	return &columnDataTypes, err
+	return &columnDataTypes, columnNamesSorted, err
 }
 
 // Generate go struct entries for a map[string]interface{} structure
-func generateMysqlTypes(obj map[string]map[string]string, depth int, jsonAnnotation bool, gormAnnotation bool, gureguTypes bool) string {
+func generateMysqlTypes(obj map[string]map[string]string, columnsSorted []string, depth int, jsonAnnotation bool, gormAnnotation bool, gureguTypes bool) string {
 	structure := "struct {"
 
-	keys := make([]string, 0, len(obj))
-	for key := range obj {
-		keys = append(keys, key)
-	}
-	sort.Strings(keys)
+	//keys := make([]string, 0, len(obj))
```

Comment:
Go ahead and remove these sections since now you pass in the sorted columns

Reply:
ok, This project helped me a lot , thank you

Reply:
Thanks!